README.md

```diff
@@ -198,6 +200,44 @@ If you want to get more insights into processing those CloudEvents or even defin
 
 We have dummy cloud-events in the form of [RFC 2616](https://ietf.org/rfc/rfc2616.txt) requests in the [test-events/](test-events/) directory. These can be easily executed using third party plugins such as the [Huachao Mao REST Client in VS Code](https://marketplace.visualstudio.com/items?itemName=humao.rest-client).
 
+### Auto signoff commit messages
+
+We have a DCO check which runs on every PR to check if the commit has been signed off.
+
+we can do 
+```bash
+git commit -ammend --signoff
+```
+
+or something like
+```bash
+git rebase HEAD~2 --signoff
+```
+
+to signoff commits, But it is inconvenient. we can automate this process.
```

Comment:
```suggestion
to signoff commits, but it is inconvenient. You can automate this process.
```
I think we should `You` here because, we want the contributors to do this.